src/main/java/com/quimify/api/health/HealthResult.java

```diff
@@ -0,0 +1,48 @@
+package com.quimify.api.health;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HealthResult {
```

Comment:
Just `boolean healthy` plus `List<String> errors`

Reply:
Okey! Now only uses this.healthy, this.errors & this.message.
This.message is still needed In order to show the message: "System errors detected: " + errors

Reply:
Without `String message` is OK